server/channels/db/migrations/postgres/000110_delete_empty_drafts.up.sql

```diff
@@ -0,0 +1,2 @@
+UPDATE drafts
+SET deleteat = updateat WHERE message = ''
```

Comment:
@agnivade, is this potentially a case where a mass UPDATE (or DELETE) could cause locking issues?

Reply:
Since there is a filter here, that should be fine. What we should avoid is running `UPDATE` on the entire table, for example, when adding a new column that needs to be filled with data. 

Unless potentially _all_ rows could have message as empty? I think that would be very improbable.

Reply:
@agnivade It's definitely unlikely but could be possible if everyone just deleted all their drafts at the same time :P

Reply:
Can you clarify what situations lead to an empty message field? Is it only when users delete their drafts? Or are there others?

Reply:
I think that's the only situation I can think of

Reply:
Let me run some queries on Community to verify the state.

Reply:
Ok, so for community 83% of the rows are empty. That's a whole lot IMO, so I would be leaning towards making this a batch migration. This will result in the migration becoming a lot more complicated now, but I don't see any other way if we are to uphold our "no locking migrations" objective.